gennet-module/gennet.py

```diff
@@ -127,33 +152,38 @@ def generate_config_model(ctx):
     n = ctx.params["num_nodes"]
     # degrees = nx.random_powerlaw_tree_sequence(n, tries=10000)
     degrees = [random.randint(1, n) for i in range(n)]
-    if (sum(degrees)) % 2 != 0:  # adjust the degree to be even
+    if (sum(degrees)) % 2 != 0 :  # adjust the degree to be even
```

Comment:
Why there are a lot of those spaces?

Reply:
the PEP8 style changes are pushed to the repo.